<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutUsWidgetsClass</class>
 <widget class="QWidget" name="AboutUsWidgetsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1548</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AboutUsWidgets</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>130</y>
     <width>861</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>75</y>
      <width>511</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>软件发布版本：V1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>861</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>————————————————————————————————————————————————————————————————————————</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>21</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>发布版本</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>290</y>
     <width>861</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>80</y>
      <width>541</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>软件完整版本：V1.0.0.2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>20</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>完整版本</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>861</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>————————————————————————————————————————————————————————————————————————</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>460</y>
     <width>861</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>21</x>
      <y>90</y>
      <width>541</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>主控板硬件版本号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>主控板硬件、软件版本号</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>861</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>————————————————————————————————————————————————————————————————————————</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>541</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>主控板软件版本号</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>541</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>主控板版本号</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
